extensions/cornerstone/src/commandsModule.ts

```diff
@@ -556,9 +555,13 @@ function commandsModule({
         return actorEntry.uid.includes(displaySetInstanceUID);
       });
 
-      const { actor: volumeActor } = actorEntry;
+      const { actor: volumeActor, uid: volumeId } = actorEntry;
 
-      setColormap(volumeActor, colormap);
+      // viewport.setProperties({ colormap }, volumeId);
+      const mapper = volumeActor.getMapper();
+      mapper.setSampleDistance(1.0);
```

Comment:
Can you add a comment about what this is doing?  It feels really opaque to me why you added this code.

Reply:
nothing removed